### ISSUE TITLE
[spike] Investigate jsx-a11y/role-supports-aria-props ignore on Tabs

### ISSUE THREAD

**Issue by mjhenkes** [cerner/terra-framework#1181]
# Bug Report

## Description
In order to quickly get back to a passing CI build we have set the [_TabRollup.jsx](https://github.com/cerner/terra-framework/blob/main/packages/terra-tabs/src/_TabMenu.jsx) file to ignore jsx-a11y/role-supports-aria-props. We need to investigate if this is a problem and how to correct it.

## Steps to Reproduce

1. Remove the jsx-a11y/role-supports-aria-props ignore from _TabRollup.jsx
2. Run eslint
3.
4.

## Expected Behavior
No failures and an accessible tab control

## Possible Solution
>You'll want to look into as least switching the role to "button", but i think the proper fix should take into consideration the "menubutton" role, and that would involve corresponding changes to the PopupMenu and PopupMenuListItem. A guide can be found here:
https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html

\- [Dave Kasper](https://github.com/cerner/terra-framework/pull/1166#discussion_r448451433)

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):
